src/app/settings/user-subscription.component.html

```diff
@@ -42,7 +42,6 @@ <h1>
       bit-button
       buttonType="secondary"
       #reinstateBtn
-      type="button"
```

Comment:
Any reason for removing this? We should always specify `type` for buttons, and `type="button"` is a good default. Otherwise it can default to a submit button and cause unexpected behaviour.

Reply:
Same comment for all other buttons on this page where you've removed the `type` attribute.

Reply:
We should omit `type="button"` on `button` elements unless it's in a form, in which case we need type to not make it a submit button.

Reply:
That seems reasonable, it is form behaviour that I was concerned about. I couldn't find the PR you refer to, what's the accessibility reason? Is it read out by screenreaders or something?

Reply:
Seems to be my bad. I had confused `role` and `type`. Let's keep `type="button"` to be safe.

Reply:
Added `type="button"`back to the button declarations